runtime/themes/catpuccin.toml

```
@@ -0,0 +1,93 @@
+attribute = "blue"
+keyword = "blue"
+"keyword.directive" = "red" # -- preprocessor comments (#if in C)
+namespace = "peach"
+punctuation = "white"
+"punctuation.delimiter" = "blue"
+operator = "blue"
+special = "peach"
+variable.other.member = "green"
```

```suggestion
"variable.other.member" = "green"
```

This was recently fixed in the default theme too: https://github.com/helix-editor/helix/commit/682bcc3a32d90bfb58872610af20f47ee4089c83

---

oh!